tests/berlin/test_state_transition.py

```diff
@@ -27,7 +27,7 @@
 ETHEREUM_TESTS_PATH = TEST_FIXTURES["ethereum_tests"]["fixture_path"]
 
 # Run state tests
-test_dir = f"{ETHEREUM_TESTS_PATH}/BlockchainTests/"
+test_dir = f"{ETHEREUM_TESTS_PATH}/LegacyTests/Cancun/BlockchainTests/"
```

Comment:
Just confirming this should be "Cancun" in the berlin fork.

Reply:
Yes. `Frontier` to `Constantinople` are under `/LegacyTests/Constantinople` and then `Istanbul` to `Shanghai` are under `/LegacyTests/Cancun`